drivers/spi/Kconfig.sam

```diff
@@ -13,86 +14,126 @@ menuconfig SPI_SAM
 config SPI_SAM_PORT_0
 	bool "Enable SPI0"
 	depends on SPI_SAM
+	select SPI_0
 	help
 	  Enable SPI0 at boot
 
 if SPI_SAM_PORT_0
 
-config SPI_SAME70_PORT_0_PIN_CS0
+choice SPI_SAM_PORT_0_PIN_CS0
 	bool "CS0 pin"
+	optional
 	depends on SOC_SERIES_SAME70
 
-choice SPI_SAME70_PORT_0_PIN_CS1
+	config SPI_SAM_PORT_0_PIN_CS0_PB2
+		bool "PB2"
+		depends on SOC_SERIES_SAME70
+
+endchoice
+
+choice SPI_SAM_PORT_0_PIN_CS1
 	bool "CS1 pin"
 	optional
 	depends on SOC_SERIES_SAME70
 
-	config SPI_SAME70_PORT_0_PIN_CS1_PA31
+	config SPI_SAM_PORT_0_PIN_CS1_PA31
```

Comment:
I think this might be better done in device tree instead of kconfig. @galak?

Do all SoCs in the e70 series have the same CS pin options?

Reply:
Yes, currently they have this configuration + dts_fixups. My intention is at least enable the configuration for others SoC families and keep minimal changes for now and then refactor.